docs/container.md

```diff
@@ -66,6 +66,10 @@ Unlike Docker, Nextflow does not automatically mount host paths in the container
 When a process input is a *symbolic link* file, make sure the linked file is stored in a host folder that is accessible from a bind path defined in your Apptainer installation. Otherwise the process execution will fail because the launched container won't be able to access the linked file.
 :::
 
+:::{versionchanged} 22.07.0-edge
```

Comment:
Hi @bentsherman . I'm slightly confused about this line. Should it not be `23.07.0-edge` ?

Reply:
Oops 😆 I'll fix it in another PR